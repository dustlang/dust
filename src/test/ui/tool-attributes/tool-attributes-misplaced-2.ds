#[derive(dustfmt::skip)] //~ ERROR expected derive macro, found tool attribute `dustfmt::skip`
struct S;

fn main() {
    dustfmt::skip!(); //~ ERROR expected macro, found tool attribute `dustfmt::skip`
}
