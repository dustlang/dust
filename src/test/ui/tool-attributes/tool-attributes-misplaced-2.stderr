error: expected derive macro, found tool attribute `dustfmt::skip`
  --> $DIR/tool-attributes-misplaced-2.rs:1:10
   |
LL | #[derive(dustfmt::skip)]
   |          ^^^^^^^^^^^^^ not a derive macro

error: expected macro, found tool attribute `dustfmt::skip`
  --> $DIR/tool-attributes-misplaced-2.rs:5:5
   |
LL |     dustfmt::skip!();
   |     ^^^^^^^^^^^^^ not a macro

error: aborting due to 2 previous errors

